File: go/src/fizzbuzz/fizzbuzz.go
package fizzbuzz

func FizzBuzz(number int) (string, bool) {
	if number%3 == 0 && number%5 == 0 {
		return "FizzBuzz", true
	} else if number%3 == 0 {
		return "Fizz", true
	} else if number%5 == 0 {
		return "Buzz", true
	} else {
		return nil, false
	}
}

File: elixir/fizzbuzz/lib/fizzbuzz.ex
defmodule Fizzbuzz do

  def fizzbuzz(number) do
    cond do
      rem(number, 3) == 0 && rem(number, 5) == 0 -> "FizzBuzz"
      rem(number, 3) == 0 -> "Fizz"
      rem(number, 5) == 0 -> "Buzz"
      true -> nil
    end
  end
end
